lib/functions/ProofUtil.ts

```diff
@@ -35,13 +53,23 @@ function partiallyValidateJWS(jws: string): void {
   }
 }
 
-function setJWSDefaults(args: JWTSignerArgs, issuerUrl: string, clientId?: string): JWTSignerArgs {
+function setJWSDefaults(
```

Comment:
Suggest to rename this to something like createJWT. Because it is certainly not settings defaults on a JSON Web Signature

Reply:
renamed to `createJWT`